.gitlab-ci.yml

```diff
@@ -64,6 +64,8 @@ test:acceptance:
     - apk add git
     - git submodule sync --recursive
     - git submodule update --init --recursive
+    # Log in to pull test image from registry.gitlab.com
+    - echo $CI_REGISTRY_PASSWORD | docker login -u $CI_REGISTRY_USER $CI_REGISTRY --password-stdin
```

Comment:
@oldgiova Is this the right way to log in? I don't remember what did we agree on :pray: